go.mod

```
@@ -0,0 +1,12 @@
+module github.com/enterprise-contract/gogather
```

```suggestion
module github.com/enterprise-contract/go-gather
```

So when folk import they can use `gather.Something`